### ISSUE TITLE
Midterm peer review

### ISSUE THREAD

**Issue by Yuanping919** [saaqebs/spotify-songs#10]
### 3 things I like:
1, The description of the data is very clear and well-structures including how they clear the data, data types of each columns. And the visualization provides readers with a straightforward insight into the data.
2, I am amazed at their high R-square and low MSE both on the training set and especially on their test set. What’s more they also implement the F-test to see whether the explanatory variables are joint significant, which shows their strictness towards this model.
3, It’s worth mentioning that this group has done very deep analysis about their model. For example, they try to investigate the effectiveness excluding the “year” variable. The techniques they take could create a greater and in-depth understanding of the dataset and opportunities to lower the errors.
### Suggestions on this report:
1, I think it could be better if they can include some explanations on each variable at the very beginning. Maybe one sentence is enough. This can help readers more clearly capture what intuition is behind this report especially for those who don’t know music a lot.
2, Discrepancy exists in the labeling of the figure. It seems this is also mentioned in some other review suggestions.
3, Maybe this report could include more explanatory variables. Because there are so many datapoints, we don’t need to worry about the overfitting problem as we do in the homework.
